Releases.md

```diff
@@ -19,6 +20,14 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 | [1.1.0](#directml-110) | [DML_FEATURE_LEVEL_2_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_2_0) | Version 2004 (Build 10.0.19041; 20H1, "May 2020 Update") | -                                                                                         |
 | [1.0.0](#directml-100) | [DML_FEATURE_LEVEL_1_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_1_0) | Version 1903 (Build 10.0.18362; 19H1, "May 2019 Update") | -                                                                                         |
 
+# DirectML 1.9.1
+
+- Fixed bug in OneHot operator when using large uint64 indices. 
```

Comment:
>OneHot [](http://example.com/codeflow?start=15&length=6)

Elsewhere in this document we format these operator names as they appear in the DirectML.h header, so this should be DML_OPERATOR_ONE_HOT (same comment applies to "Join" below)